endpoint/endpoint.go

```diff
@@ -58,5 +85,9 @@ func (e *Endpoint) MergeLabels(labels map[string]string) {
 }
 
 func (e *Endpoint) String() string {
-	return fmt.Sprintf(`%s -> %s (type "%s")`, e.DNSName, e.Target, e.RecordType)
+	var ttl string
+	if e.RecordTTL.IsConfigured {
+		ttl = fmt.Sprintf(" TTL: %v", e.RecordTTL.Value)
+	}
+	return fmt.Sprintf(`%s -> %s (type "%s")%s`, e.DNSName, e.Target, e.RecordType, ttl)
```

Comment:
I would like to have this changed from the original code and use a DNS config like string:

    return fmt.Sprintf(`%s %d IN %s %s `, e.DNSName, ttl, e.RecordType, e.Target)

srcname 60 IN CNAME destination